src/app/not-found/not-found.component.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html lang="en" style="height: 100%">
+  <head>
+    <meta charset="utf-8"/>
+    <title>MifosX - 404</title>
+    <base href="/"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"/>
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <meta name="mobile-web-app-capable" content="yes">
+
+    <link rel="icon" type="image/x-icon" href="favicon.ico"/>
+    <link rel="icon" type="image/png" href="assets/images/MifosX_logo.png" sizes="182x184">
+    <link rel="apple-touch-icon" type="image/png" href="apple-touch-icon.png" sizes="182x184"/>
+    <link rel="manifest" href="manifest.json"/>
+
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Mono:300" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
+  </head>
+  
+  <body>
```

Comment:
Jonathan, as I told you earlier remove the code from lines 1-21 and 27-28, a component only contains part of html code, which is loaded in place of the `<mifosx-web-app>` selector present in `index.html`, so these parts are not supposed to be present here. These parts are already there in `index.html`.